### PR TITLE
Add varlink support for prune

### PR DESCRIPTION
**Pull request by baude**
Add the ability to prune unused images using the varlink
API.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
@baude Lint errors

**Comment by rhatdan** [comment]
LGTM
@mheon @umohnani8 @vrothberg @giuseppe PTAL

File: pkg/varlinkapi/images.go [hunk 1]
```diff
@@ -620,3 +620,21 @@ func (i *LibpodAPI) ContainerRunlabel(call iopodman.VarlinkCall, input iopodman.
...
+	var (
+		pruned []string
+	)
+	pruneImages, err := i.Runtime.ImageRuntime().GetPruneImages()
+	if err != nil {
+		return err
+	}
+	for _, i := range pruneImages {
+		if err := i.Remove(true); err != nil {
+			return call.ReplyErrorOccurred(err.Error())
```
Comment (vrothberg) [inline pkg/varlinkapi/images.go]: We're wrapping the error in the native client via `errors.Wrapf(err, "failed to remove %s", i.ID())` (absolutely non-blocking).

**Comment by vrothberg** [comment]
(I counted Dan's LGTM)

**Pull request merged**